codegen.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Is it absolutely necessary to have this in another file?

We try to avoid having configuration files at the root of the project unless strictly necessary.
I suppose it is at the size where you wouldn't want it in `package.json`

Reply:
yes that was my reasoning.

Reply:
I was also not sure if all and any scalars would be represented as strings.
If that is the case, I could change `"defaultScalarType": "unknown",` to `"defaultScalarType": "string",` and avoid the custom declarations for every single type.

That would mean that this config could be added to the `package.json` again.

Reply:
is it ok for you to have the config of this size in the `package.json`?
If so I will move it back in.